modules/local/filter_assembly.nf

```diff
@@ -0,0 +1,52 @@
+process FILTER_ASSEMBLY {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/seqkit:2.8.1--h9ee0642_0':
+        'biocontainers/seqkit:2.8.1--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(assembly)
+
+    output:
+    tuple val(meta), path("${prefix}_filtered.fasta.gz") , emit: fastx
```

Comment:
should the emit be called `fasta` if the output is always a fasta?

Reply:
yeah.. good point. I forgot to change that